name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SolarisbankAGPDFExtractor.java

```diff
@@ -81,7 +81,7 @@ private BiConsumer<AccountTransaction, ParsedData> assignmentsProvider()
             transaction.setDateTime(asDate(matcherMap.get(CONTEXT_KEY_DATE)));
             transaction.setAmount(asAmount(matcherMap.get(CONTEXT_KEY_AMOUNT)));
             transaction.setCurrencyCode(matcherMap.get(CONTEXT_KEY_CURRENCY));
-            transaction.setNote(matcherMap.get(CONTEXT_KEY_NOTE));
+            transaction.setNote(matcherMap.get(CONTEXT_KEY_NOTE).trim());
```

Comment:
change to 
`transaction.setNote(trim(matcherMap.get(CONTEXT_KEY_NOTE)));`

look [here](https://github.com/buchen/portfolio/blob/777cd54cffb05dbd1493eb03b2d026c56098ffdd/name.abuchen.portfolio/src/name/abuchen/portfolio/util/TextUtil.java#L71:L90)